app/models/website.rb

```diff
@@ -12,7 +12,8 @@ def update_puppies
       name      = dog[:name]
       image_url = dog[:image_url]
 
-      puppies.where(name: name).first_or_create(image_url: image_url)
+      puppy = puppies.where(name: name).first_or_create(image_url: image_url)
+      puppy.update(description: dog[:description])
```

Comment:
Since we're updating each time now, you could move ``image_url`` down to the update line:
```
puppy = puppies.where(name: name).first_or_create
puppy.update(description: dog[:description], image_url: image_url)
```

Also.. it's a bit weird that we're converting the variables from the has on line 12 & 13 but not on 16.  Maybe do a ``description = dog[:description]`` on line 14 and then change that on 16?